foundation/foundation-mda/src/main/resources/templates/cdi.container.factory.java.vm

```diff
@@ -58,9 +59,10 @@ public final class CdiContainerFactory {
     protected static List<CdiContext> getContexts() {
         List<CdiContext> contexts = new ArrayList<>();
         contexts.add(new PipelinesCdiContext());
-        ## TODO: would be nice to only include if we need messaging, but right now we assume messaging everywhere
+## TODO: would be nice to only include if we need messaging, but right now we assume kafka messaging everywhere
```

Comment:
I: Space before `## TODO` was being rendered in downstream template, resulting in the spacing being off in the following line.